js/main.js

```diff
@@ -10,3 +10,85 @@ navLinks.forEach((link) => {
     document.body.classList.remove("show-links");
   });
 });
+
+// model popup
+const data = {
+  name: "Keeping track of hundreds of components",
+  description:
+    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  imageDesktop: "images/desktop-image-portfolio.png",
+  githubIcon: "images/icongithub.svg",
+  liveIcon: "images/icon-export.svg",
+  technologiesMobile: ["Ruby on Rails", "CSS", "JavaScript"],
+  technologiesDesktop: [
+    "CodeKit",
+    "GitHub",
+    "JavaScript",
+    "Bootstrap",
+    "Terminal",
+    "Codepen",
+  ],
+  sourceLink: "https://github.com/MohamedHNoor/mobile-firs-project1",
+  liveLink: "https://mohamedhnoor.github.io/Portfolio-project1/",
+};
```

Comment:
- [ ] Kindly make sure the JavaScript code implements an array of objects to store JavaScript objects containing the information for each project and not only for one project. i.e. you should repeat the same approach as you did above for each project object and all objects should be wrapped in an `Array`.
e.g.:
```javascript
const projects = [
  {
    id: "1",
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
    link: "link_to_source_version.com",
},
 {
    id: "2",
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
    link: "link_to_source_version.com",
},
...
]
```